tests/operators/test_latest_only_operator.py

```diff
@@ -47,15 +47,40 @@ def get_task_instances(task_id):
 
 
 class LatestOnlyOperatorTest(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        from tests.compat import MagicMock
+        from airflow.jobs import SchedulerJob
 
-    def setUp(self):
-        super(LatestOnlyOperatorTest, self).setUp()
-        self.dag = DAG(
+        cls.dag = DAG(
```

Comment:
This fix is needed for the test because the original test calls `LatestOnlyOperator.execute()` without first creating a dagrun. Also the `create_dag_run` is moved to `setUpClass` so that it's called only once for the whole class.